inc/macro_utils/macro_utils.h

```diff
@@ -209,6 +209,12 @@ const char* MU_C3(MU_, enumIdentifier,_ToString)(enumIdentifier value)
 
 #define MU_ENUM_TO_STRING_2(enumIdentifier, value) MU_C3(MU_, enumIdentifier,_ToString)(value)
 
+/*PRI_BOOL is the format specifier that prints a bool/_Bool*/
+#define PRI_BOOL "s (%d)"
+
+/*MU_BOOL_VALUE is the counterpart of PRI_BOOL*/
+#define MU_BOOL_VALUE(v) ((const char*[]){"false", "true"})[!!v], (v)
```

Comment:
Such an interesting construct. My compiler spent a bit of time compiling :-). But it looks really nice, specially since it is encapsulated here.

Reply:
thanks